examples/selectvar.c

```diff
@@ -7,6 +7,7 @@
 #include <memory.h>
 #include <snowflake/client.h>
 #include <example_setup.h>
+#include "string.h"
```

Comment:
Is this external library? then use
```
#include <string>
```

Reply:
This is a C header for strcpy. I don't understand why master branch did not have problem. I did add anything into this test.

Any, it should be 
```
#include <string.h>
```